src/storage/heap_file.c

```diff
@@ -25304,6 +25388,16 @@ heap_update_set_prev_version (THREAD_ENTRY * thread_p, const OID * oid, PGBUF_WA
 
       error_code = or_mvcc_set_log_lsa_to_record (&forward_recdes, prev_version_lsa);
 
+      if (heap_ovf_find_vfid (thread_p, hfid, &ovf_vfid, false, PGBUF_UNCONDITIONAL_LATCH) == NULL)
+	{
+	  assert (false);
+	  error_code = ER_FAILED;
+	  goto end;
+	}
+
+      log_append_redo_data2 (thread_p, RVHF_SET_PREV_VERSION_LSA, &ovf_vfid, overflow_pg_watcher.pgptr, -1,
```

Comment:
Nit (all new occurences):
```suggestion
      log_append_redo_data2 (thread_p, RVHF_SET_PREV_VERSION_LSA, &ovf_vfid, overflow_pg_watcher.pgptr, NULL_OFFSET,
```

Reply:
Thanks. I have also changed to use NULL_OFFSET in the recovery function.